src/services/MovieService.js

```diff
@@ -24,6 +24,14 @@ class MovieService extends ApiService {
   incrementVisits = (id) => {
     return this.apiClient.patch(ENDPOINTS.MOVIES + id + '/visits/');
   };
+
+  postComment = (payload) => {
+    return this.apiClient.post(ENDPOINTS.MOVIES + payload.movie + '/post-comment/', {comment: payload.comment});
+  };
+
+  getComments = (payload) => {
+    return this.apiClient.get(ENDPOINTS.MOVIES + payload.movie + '/comments/?page=' + payload.page);
+  };
```

Comment:
sad sam obratio malo vise paznju na ove endpoint-ove i po ovom sad, gubi nam smisao onaj gornji endpoint koji si samo definisao. Poenta endpoint-a je da imas citav endpoint ddefinisan i samo zamjenis stvari sa realni parametrima. To mozes posticit za funkcjiom `format from 'util'`, util je nesto sto posotji na nivou projekta odmah, tako da dovoljno je samo da importujes `{format} from 'util'`. Evo doukmentacije https://nodejs.org/api/util.html#util_util_format_format_args.

I onda implementacija za endpoint-e bi bila ovako nesto
```
ENDPOTINS: {
POST_MOVIE: '/movies',
DELETE_MOVIE: '/movies/%d',
GET_VISITS: '/movies/:d/visits'
 ...
}

a onda dole koristis 

...get(format(ENDPOINT.VISITIS, movieId))
```